lib/train/platforms/detect/os_common.rb

```diff
@@ -0,0 +1,55 @@
+# encoding: utf-8
+
+require 'train/platforms/detect/os_linux'
+require 'train/platforms/detect/os_windows'
+require 'rbconfig'
+
+module Train::Platforms::Detect
+  module OSCommon
+    include Train::Platforms::Detect::Linux
+    include Train::Platforms::Detect::Windows
+
+    def rbconfig(regex)
```

Comment:
I don't like this name as this will be part of what you can use in the detect logic. Maybe "get_host_from_ruby" or something?

Reply:
How about `rbconfig_host_os` or `ruby_host_os`? Not a fan of `get_` methods personally.

Reply:
i like `ruby_host_os`